src/bento/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+// These extensions have been modified in the last 2 weeks, so we're not
+// including them yet to minimize disruption to devs actively working on them.
+const EXCLUDED_EXTENSIONS = require('./import-order-excluded.json');
+
+module.exports = {
+  'rules': {'import/order': 2},
```

Comment:
nit: `import/order` is enabled for everything in `src/`, so you only need the exclusion override here
```suggestion
```

Reply:
Good catch, I missed this when I moved `/bento` to `src/bento`